ui/packages/catalog-ui-search/src/main/webapp/js/Common.js

```diff
@@ -265,6 +265,15 @@ define([
                 this.wrapMapCoordinates(lon, [-180, 180]),
                 this.wrapMapCoordinates(lat, [-90, 90])
             ]);
+        },
+        getArrayFromValue: function(value) {
+          if (value === undefined || value === '' || value === null) {
+            return []
+          } else if (Array.isArray(value)) {
```

Comment:
❓ can't these two conditions be collapsed to something like `return (_.isArray(value) && value) || [value]`?

Reply:
Then you could get `[null]` and `[undefined]` both of which would be bad.

Reply:
...that would be caught by the first predicate. I'm talking about the last two

Reply:
Though it does work, I personally find the ternary operator more readable than `&&` and `||` in the context of returning non-boolean values:
```
return _.isArray(value) ? value : [value]
```

I also don't know where we fall on `_.isArray` versus `Array.isArray`.